src/utilities/AlertHelper.jsx

```diff
@@ -7,7 +7,7 @@ export const getAlertMessage = (props) => {
 	if (hasPasswordReset) {
 		return <Note className='bc_alert alert-success'>{props.values.SignInPage.ResetPasswordAlert.replace('{email address}', props.history.location.state)}</Note>
 	}
-	else if (props.status) 
+	else
```

Comment:
you don't even need an else here.

if hasPassword is true, the function will return. If its false, then just run the other logic.

Reply:
fixed